.github/workflows/release.yml

```diff
@@ -14,12 +14,48 @@ concurrency:
   cancel-in-progress: true
```

Comment:
I think we can all delete the draft release action.
We wont need it with semantic release.